docs/samples/transformer/image_transformer/README.md

```diff
@@ -47,7 +52,7 @@ class ImageTransformer(Transformer):
 ### Build Transformer docker image
 This step can be part of your CI/CD pipeline to continuously build the transformer image version. 
 ```shell
-docker build -t yuzisun/image-transformer:latest -f transformer.Dockerfile .
+docker build -t pugang/image-transformer:latest -f transformer.Dockerfile .
 ```
```

Comment:
@ellis-bigelow can we establish an official docker repo for kfserving? I dont like individual github ids

Reply:
those are for CI images, I am also thinking about using these examples for e2e tests.

Reply:
@pugangxa we can build this image in CI and publish to `gcr.io/kubeflow-ci/kfserving/image-transformer`, we can use it for both e2e test and example.

Reply:
Our current official release of tagged kfserving images is gcr.io/kfserving. If we need to publish images there, you can add it to https://github.com/kubeflow/kfserving/tree/master/release/triggers

Reply:
Thanks Yuzi & Ellis. 
Take a look at both the gcr.io/kfserving for official build and gcr.io/kubeflow-ci/kfserving/ for CI. I think the image here is suitable to the CI location since it's just a sample/e2e image. 
/cc @jinchihe , would you please help add this image build to gcr.io/kubeflow-ci/kfserving/image-transformer? To be honest I'm not familiar with it and don't know how to do. After it's in e2e step, in this PR and PR #525 we will update to use the new location of gcr.io/kubeflow-ci/kfserving/image-transformer for image pull. 

Regards
Gang

Reply:
@pugangxa OK, agree with @yuzisun , we can try to build this in the CI build image step.